### DISCUSSION TITLE
Account lock out time after too many failed password attempts

### DISCUSSION THREAD

**Discussion by NUMarkMelone** [community/community#47019] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What is the default lock out time after too many failed password attempts?

**Comment by jge162** [comment]
I do not know where you can find a lock out time, you need to reach out to support to be honest. We can only discuss limited stuff here. Here is link to [support](https://support.github.com/request/account?tags=hubberfy_account)  choose the option `sign in issues.`

**Comment by NUMarkMelone** [comment]
It appears that the default lock out time is 15 minutes.

**Comment by bari444** [comment]
why i cannot just get loggin link via my email lol

**Reply by mike-code** [reply]
I guess it wasn't only me who got locked out today

**Comment by svidlak** [comment]
Not sure why thats even a thing, but at least write the cooldown time, or send an email with an OTP code, or at least an email that will alert me about failed attempts and warn me about the account lock

**Reply by drescherjm** [reply]
I just got an email about the 2FA failure but it didn't mention the length of the lockout period. In my case I was attempting to logon in a VM and couldn't easily connect my hardware key and I didn't have a current authenticator app setup. Then after setting up the app on my device that has the hardware key I typed the passcode in the wrong 2FA method screen a few times before I saw it. This locked me out of the vm.

**Comment by Alexander96f** [comment]
yep this is horrible just locked myself out in the worst possible time just sned a fucking email code
